builder/Dockerfile

```diff
@@ -6,8 +6,12 @@ RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor github.com/Azure/aks-periscope
 
 FROM alpine
 RUN apk --no-cache add ca-certificates
+RUN apk add --no-cache curl openssl bash
 ADD https://storage.googleapis.com/kubernetes-release/release/v1.16.0/bin/linux/amd64/kubectl /usr/local/bin/kubectl
 RUN chmod +x /usr/local/bin/kubectl
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \

```

Comment:
this step adds about 50MB to the final periscope image size (100MB originally). Ideally we would have separate images for helm vs. non helm but this is simpler. Potentially revisit later

Reply:
Sounds good.